### DISCUSSION TITLE
[ERROR] Bundle PimcoreDataHubBundle can not be installed

### DISCUSSION THREAD

**Discussion by vinitchopra12** [pimcore/pimcore#9585] [Q&A / Support]
I am getting this error while install the DataHubBundle.

![image](https://user-images.githubusercontent.com/14720025/123457928-11b0a500-d602-11eb-866c-d5ccba98fa28.png)

Below are the configuration I have:
Pimcore v10.0.6
PHP 8.0.6

paragonie/constant_time_encoding (v2.4.0)
 phpseclib/phpseclib (2.0.32)

**Comment by vinitchopra12** [comment]
Any update on this?

**Comment by Rojal123245** [comment]
composer require pimcore/data-hub -W
worked for me

**Reply by vinitchopra12** [reply]
Till enable the bundle everything works fine, but giving me an error while running pimcore install bundle command.

**Comment by ShaileshT-dot** [comment]
Hi, Can you post errors from log file? (Log file can be found in var/log folder. File can be dev.log/prod.log and php.log)

**Reply by vinitchopra12** [reply]
Hi, 
I see only dev.log file is generated after I run the install command. PFA, the dev.log file

[dev.log](https://github.com/pimcore/pimcore/files/6741721/dev.log)

Also, due to this issue DataImporter Option is not visible in Pimcore Admin UI.

**Comment by dvesh3** [comment]
@vinitchopra12 please check if the bundle is already installed:
![image](https://user-images.githubusercontent.com/5137917/123982324-37072f80-d9c3-11eb-863a-7da64bc72fdc.png)

**Reply by vinitchopra12** [reply]
Yes, it shows the Bundle is installed, but not created any tables in Database and not showing the DataImporter Option in Admin. 
![image](https://user-images.githubusercontent.com/14720025/123983659-ac81f880-d9e1-11eb-9fb4-5fa9d5b44d3c.png)

![image](https://user-images.githubusercontent.com/14720025/123984166-0da9cc00-d9e2-11eb-909a-fb0df7315433.png)

**Reply by dvesh3** [reply]
check console for any errors on opening datahub window

**Reply by vinitchopra12** [reply]
Showing me some error:

![image](https://user-images.githubusercontent.com/14720025/123988475-aee65180-d9e5-11eb-9f76-7e4bfb6333dd.png)

**Reply by dvesh3** [reply]
I guess you have installed DataImporter before DataHub Bundle...so just try one simple tweak. open `/var/config/extensions.php` and make sure Dataimporter bundle entry is placed after DataHub.
![image](https://user-images.githubusercontent.com/5137917/123990761-5b1a3f00-d9ca-11eb-9330-76a6537e032c.png)

**Reply by vinitchopra12** [reply]
I did the suggested tweak, DataImporter is showing now. 
But still the Datahub Install command is not working and not creating the tables in database also. Due to this when I create any configuration in Dataimporter or Datahub it throws an error as:

![image](https://user-images.githubusercontent.com/14720025/123997902-831b9980-d9ee-11eb-9c6e-fc956a3aed54.png)

**Reply by dvesh3** [reply]
Seems like something went wrong while installing the datahub bundle...please run the db query ``DELETE FROM `users_permission_definitions` WHERE `key` = 'plugin_datahub_config';`` and install the bundle again.
